docs/user_documentation.md

```diff
@@ -783,6 +783,45 @@ The following are fields on the Data Source Object:
 
 -   **Reset Data Source (Console Button):** Disables and resets data source to retrieve data from specified Start Date (Depending on import profile settings, enabling this data source after a reset could duplicate data in the Workspace)
 
+### Data Source Specific Settings
+
+This section contains additional settings which are not associated with specific
+Relativity Fields. The settings described here are common across all Data Source
```

Comment:
I don't understand this sentence. Can you explain what you were trying to get here?

Reply:
I'm not sure which sentence you mean:

#788 This section contains additional settings which are not associated with specific
Relativity Fields

The fields above (Name, Data Source Type, ... , Reset Data Source Console button) exist as actual Relativity field objects, whereas the ones in `Data Source Specific Settings` do not correspond 1:1 to Relativity fields. They are stored in a `Custom Feilds Store` and parsed out by some JavaScript on the front end.

They are separated by section because they have separate sections in the Data Source view and because they require slightly different wording to explain.

#789  The settings described here are common across all Data Source...

This is kind of a strange concept to document, but these fields exist on the DataTypeSettings object (which means that EVERY data source will have the fields), but it's possible to add fields to specific data source types. For example, in an Exchange data source you can set a target data batch size per user in addition to a total target size.

The idea is to document the common fields in a single place so we can reference them below. Previously we were only documenting them on Relativity Native Data Extraction Data Source Types, which was weird.